### DISCUSSION TITLE
The GitHub API cannot trigger workflow in certain repo

### DISCUSSION THREAD

**Discussion by DonghaoFeng-Moloco** [community/community#36592] [Actions]
The GitHub API cannot trigger the workflow when I set the workflow condition to workflow_dispatch for a certain repo. But it can be triggered if the file is in other repos. I have checked the setting of other repos, which has the same setting with the repo I am working on. The response message is "404 not found". I cannot find the reason for that. 

Anyone can help? I am really appreciate it!

**Comment by ossanna16** [comment]
Hey there @DonghaoFeng-Moloco and thank you for asking a good question! I'm glad you found our community 🙂
